src/components/AllocationOverview.js

```diff
@@ -231,9 +231,9 @@ const AllocationOverview = (props) => {
     const editButtonDisabled = (
         updatedAllocationRate.total_amount == allocationInfo.amount &&
         updatedAllocationRate.hourly_rate == allocationInfo.rate.hourly_rate &&
-        updatedAllocationPayment?.id == allocationInfo.payment?.id
+        updatedAllocationPayment?.id == allocationInfo.payment?.id &&
+        updatedAllocationEndDate == allocationInfo.end_date
```

Comment:
Don't we also need to add `start_date` to this condition? @mecoccaro 

the test case didn't show when only the start date changes...

If everything except the start date is the same it seems this code would still disable the button right? Which is not the desired behavior. Am i missing something?

Reply:
As it is right now, when you modify either start or end date the button will be enable because the calendar functionality changes the entire date, no matter if you just selected the start or end date.